tests/AppBundle/Controller/Admin/BlogControllerTest.php

```diff
@@ -84,9 +84,9 @@ public function testBackendHomepage()
 
         $crawler = $client->request('GET', '/en/admin/post/');
 
-        $this->assertCount(
-            30,
-            $crawler->filter('body#admin_post_index #main tbody tr'),
+        $this->assertGreaterThanOrEqual(
+            1,
+            $crawler->filter('body#admin_post_index #main tbody tr')->count(),
```

Comment:
Ooh! `GreaterThanOrEqual` you're right, this answers my first comment too (`Why do you need that the first Post belongs to Jane`)